lms/templates/problem_notifications.html

```diff
@@ -4,9 +4,10 @@
 from openedx.core.djangolib.markup import HTML
 %>
 
-<div class="notification ${notification_type}" tabindex="-1">
+<div class="notification ${notification_type} ${'notification-hint' if hint_notification else 'notification-response'}" tabindex="-1">
```

Comment:
No. The Save notification and a response notification do not show at the same time. Once Submit is clicked, the save message will be gone, and replaced by the submit notification.